exercises/pig-latin/.meta/src/reference/java/PigLatinTranslator.java

```diff
@@ -41,6 +47,18 @@ private String translateWord(String word) {
         return word.substring(1) + word.toCharArray()[0] + AY;
     }
 
+    private static boolean hasYAfterConsonantCluster(String word) {
+        for(int i = 0; i < word.length() - 1; i++){
```

Comment:
There should be white space after `for` and before `{`. And please try to keep the indentation consistent, everything should be indented with four spaces like the rest of the file 🙂